error: 
    status - 500 
    message - ValidationError: "name" is required 
    url - /api/v1/users/ 
    method - POST 
    IP - ::1
   {"timestamp":"2022-07-18T03:02:02.952Z"}
error: 
    status - 500 
    message - ValidationError: "name" is required 
    url - /api/v1/users/ 
    method - POST 
    IP - ::1
   {"timestamp":"2022-07-18T03:03:01.934Z"}
error: 
    status - 500 
    message - ValidationError: "name" is required 
    url - /api/v1/register 
    method - POST 
    IP - ::1
   {"timestamp":"2022-07-18T03:07:59.874Z"}
error: 
    status - 500 
    message - ValidationError: "name" is required 
    url - /api/v1/register 
    method - POST 
    IP - ::1
   {"timestamp":"2022-07-18T03:09:20.436Z"}
error: 
    status - 500 
    message - ValidationError: "name" is required 
    url - /api/v1/register 
    method - POST 
    IP - ::1
   {"timestamp":"2022-07-18T03:10:05.541Z"}
error: 
    status - 500 
    message - ValidationError: "name" is required 
    url - /api/v1/register 
    method - POST 
    IP - ::1
   {"timestamp":"2022-07-18T03:11:30.382Z"}
error: 
    status - 500 
    message - ValidationError: "name" length must be at least 4 characters long 
    url - /api/v1/register 
    method - POST 
    IP - ::1
   {"timestamp":"2022-07-18T03:11:57.729Z"}
error: 
    status - 500 
    message - ValidationError: "firstName" is not allowed 
    url - /api/v1/register 
    method - POST 
    IP - ::1
   {"timestamp":"2022-07-18T03:12:05.683Z"}
error: 
    status - 500 
    message - ValidationError: "name" is required 
    url - /api/v1/register 
    method - POST 
    IP - ::1
   {"timestamp":"2022-07-18T03:14:22.105Z"}
error: 
    status - 500 
    message - ValidationError: "name" is required 
    url - /api/v1/register 
    method - POST 
    IP - ::1
   {"timestamp":"2022-07-18T03:15:25.387Z"}
error: 
      status - 400
      message - Authorization token is required 
      url - /api/v1/register/register 
      method - GET 
      IP - ::1
     {"timestamp":"2022-07-18T03:45:06.007Z"}
error: 
    status - 500 
    message - ValidationError: "name" is required 
    url - /api/v1/register 
    method - POST 
    IP - ::1
   {"timestamp":"2022-07-18T03:53:02.637Z"}
error: 
    status - 500 
    message - ValidationError: "password" length must be at least 4 characters long 
    url - /api/v1/register 
    method - POST 
    IP - ::1
   {"timestamp":"2022-07-18T03:54:46.918Z"}
error: 
    status - 500 
    message - SyntaxError: Unexpected token , in JSON at position 112
    at JSON.parse (<anonymous>)
    at parse (C:\Users\kanch\OneDrive\Desktop\Registration\Login\node_modules\body-parser\lib\types\json.js:89:19)
    at C:\Users\kanch\OneDrive\Desktop\Registration\Login\node_modules\body-parser\lib\read.js:128:18
    at AsyncResource.runInAsyncScope (node:async_hooks:201:9)
    at invokeCallback (C:\Users\kanch\OneDrive\Desktop\Registration\Login\node_modules\raw-body\index.js:231:16)
    at done (C:\Users\kanch\OneDrive\Desktop\Registration\Login\node_modules\raw-body\index.js:220:7)
    at IncomingMessage.onEnd (C:\Users\kanch\OneDrive\Desktop\Registration\Login\node_modules\raw-body\index.js:280:7)
    at IncomingMessage.emit (node:events:538:35)
    at IncomingMessage.emit (node:domain:475:12)
    at endReadableNT (node:internal/streams/readable:1345:12)
    at processTicksAndRejections (node:internal/process/task_queues:83:21) 
    url - /api/v1/users 
    method - POST 
    IP - ::1
   {"timestamp":"2022-07-18T04:01:26.581Z"}
error: 
    status - 500 
    message - ReferenceError: req is not defined
    at _callee$ (C:\Users\kanch\OneDrive\Desktop\Registration\Login\src\services\/user.service.js:10:15)
    at tryCatch (C:\Users\kanch\OneDrive\Desktop\Registration\Login\node_modules\@babel\runtime\helpers\regeneratorRuntime.js:86:17)
    at Generator._invoke (C:\Users\kanch\OneDrive\Desktop\Registration\Login\node_modules\@babel\runtime\helpers\regeneratorRuntime.js:66:24)
    at Generator.next (C:\Users\kanch\OneDrive\Desktop\Registration\Login\node_modules\@babel\runtime\helpers\regeneratorRuntime.js:117:21)
    at asyncGeneratorStep (C:\Users\kanch\OneDrive\Desktop\Registration\Login\node_modules\@babel\runtime\helpers\asyncToGenerator.js:3:24)
    at _next (C:\Users\kanch\OneDrive\Desktop\Registration\Login\node_modules\@babel\runtime\helpers\asyncToGenerator.js:25:9)
    at processTicksAndRejections (node:internal/process/task_queues:96:5) 
    url - /api/v1/users 
    method - GET 
    IP - ::1
   {"timestamp":"2022-07-18T04:16:01.820Z"}
error: 
    status - 500 
    message - ReferenceError: req is not defined
    at _callee$ (C:\Users\kanch\OneDrive\Desktop\Registration\Login\src\services\/user.service.js:6:44)
    at tryCatch (C:\Users\kanch\OneDrive\Desktop\Registration\Login\node_modules\@babel\runtime\helpers\regeneratorRuntime.js:86:17)
    at Generator._invoke (C:\Users\kanch\OneDrive\Desktop\Registration\Login\node_modules\@babel\runtime\helpers\regeneratorRuntime.js:66:24)
    at Generator.next (C:\Users\kanch\OneDrive\Desktop\Registration\Login\node_modules\@babel\runtime\helpers\regeneratorRuntime.js:117:21)
    at asyncGeneratorStep (C:\Users\kanch\OneDrive\Desktop\Registration\Login\node_modules\@babel\runtime\helpers\asyncToGenerator.js:3:24)
    at _next (C:\Users\kanch\OneDrive\Desktop\Registration\Login\node_modules\@babel\runtime\helpers\asyncToGenerator.js:25:9)
    at C:\Users\kanch\OneDrive\Desktop\Registration\Login\node_modules\@babel\runtime\helpers\asyncToGenerator.js:32:7
    at new Promise (<anonymous>)
    at Object.<anonymous> (C:\Users\kanch\OneDrive\Desktop\Registration\Login\node_modules\@babel\runtime\helpers\asyncToGenerator.js:21:12)
    at Object.getAllUsers (C:\Users\kanch\OneDrive\Desktop\Registration\Login\src\services\/user.service.js:5:25)
    at _callee$ (C:\Users\kanch\OneDrive\Desktop\Registration\Login\src\controllers\/user.controller.js:12:36)
    at tryCatch (C:\Users\kanch\OneDrive\Desktop\Registration\Login\node_modules\@babel\runtime\helpers\regeneratorRuntime.js:86:17)
    at Generator._invoke (C:\Users\kanch\OneDrive\Desktop\Registration\Login\node_modules\@babel\runtime\helpers\regeneratorRuntime.js:66:24)
    at Generator.next (C:\Users\kanch\OneDrive\Desktop\Registration\Login\node_modules\@babel\runtime\helpers\regeneratorRuntime.js:117:21)
    at asyncGeneratorStep (C:\Users\kanch\OneDrive\Desktop\Registration\Login\node_modules\@babel\runtime\helpers\asyncToGenerator.js:3:24)
    at _next (C:\Users\kanch\OneDrive\Desktop\Registration\Login\node_modules\@babel\runtime\helpers\asyncToGenerator.js:25:9) 
    url - /api/v1/users 
    method - GET 
    IP - ::1
   {"timestamp":"2022-07-18T04:17:52.493Z"}
error: 
    status - 500 
    message - TypeError: Cannot read properties of undefined (reading 'body')
    at _callee$ (C:\Users\kanch\OneDrive\Desktop\Registration\Login\src\services\/user.service.js:6:48)
    at tryCatch (C:\Users\kanch\OneDrive\Desktop\Registration\Login\node_modules\@babel\runtime\helpers\regeneratorRuntime.js:86:17)
    at Generator._invoke (C:\Users\kanch\OneDrive\Desktop\Registration\Login\node_modules\@babel\runtime\helpers\regeneratorRuntime.js:66:24)
    at Generator.next (C:\Users\kanch\OneDrive\Desktop\Registration\Login\node_modules\@babel\runtime\helpers\regeneratorRuntime.js:117:21)
    at asyncGeneratorStep (C:\Users\kanch\OneDrive\Desktop\Registration\Login\node_modules\@babel\runtime\helpers\asyncToGenerator.js:3:24)
    at _next (C:\Users\kanch\OneDrive\Desktop\Registration\Login\node_modules\@babel\runtime\helpers\asyncToGenerator.js:25:9)
    at C:\Users\kanch\OneDrive\Desktop\Registration\Login\node_modules\@babel\runtime\helpers\asyncToGenerator.js:32:7
    at new Promise (<anonymous>)
    at Object.<anonymous> (C:\Users\kanch\OneDrive\Desktop\Registration\Login\node_modules\@babel\runtime\helpers\asyncToGenerator.js:21:12)
    at Object.getAllUsers (C:\Users\kanch\OneDrive\Desktop\Registration\Login\src\services\/user.service.js:5:25)
    at _callee$ (C:\Users\kanch\OneDrive\Desktop\Registration\Login\src\controllers\/user.controller.js:12:36)
    at tryCatch (C:\Users\kanch\OneDrive\Desktop\Registration\Login\node_modules\@babel\runtime\helpers\regeneratorRuntime.js:86:17)
    at Generator._invoke (C:\Users\kanch\OneDrive\Desktop\Registration\Login\node_modules\@babel\runtime\helpers\regeneratorRuntime.js:66:24)
    at Generator.next (C:\Users\kanch\OneDrive\Desktop\Registration\Login\node_modules\@babel\runtime\helpers\regeneratorRuntime.js:117:21)
    at asyncGeneratorStep (C:\Users\kanch\OneDrive\Desktop\Registration\Login\node_modules\@babel\runtime\helpers\asyncToGenerator.js:3:24)
    at _next (C:\Users\kanch\OneDrive\Desktop\Registration\Login\node_modules\@babel\runtime\helpers\asyncToGenerator.js:25:9) 
    url - /api/v1/users 
    method - GET 
    IP - ::1
   {"timestamp":"2022-07-18T04:18:21.044Z"}
error: 
    status - 500 
    message - TypeError: Cannot read properties of undefined (reading 'email')
    at _callee$ (C:\Users\kanch\OneDrive\Desktop\Registration\Login\src\services\/user.service.js:11:29)
    at tryCatch (C:\Users\kanch\OneDrive\Desktop\Registration\Login\node_modules\@babel\runtime\helpers\regeneratorRuntime.js:86:17)
    at Generator._invoke (C:\Users\kanch\OneDrive\Desktop\Registration\Login\node_modules\@babel\runtime\helpers\regeneratorRuntime.js:66:24)
    at Generator.next (C:\Users\kanch\OneDrive\Desktop\Registration\Login\node_modules\@babel\runtime\helpers\regeneratorRuntime.js:117:21)
    at asyncGeneratorStep (C:\Users\kanch\OneDrive\Desktop\Registration\Login\node_modules\@babel\runtime\helpers\asyncToGenerator.js:3:24)
    at _next (C:\Users\kanch\OneDrive\Desktop\Registration\Login\node_modules\@babel\runtime\helpers\asyncToGenerator.js:25:9)
    at processTicksAndRejections (node:internal/process/task_queues:96:5) 
    url - /api/v1/users 
    method - GET 
    IP - ::1
   {"timestamp":"2022-07-18T07:12:41.223Z"}
error: 
    status - 500 
    message - TypeError: Cannot read properties of undefined (reading 'email')
    at _callee$ (C:\Users\kanch\OneDrive\Desktop\Registration\Login\src\services\/user.service.js:10:29)
    at tryCatch (C:\Users\kanch\OneDrive\Desktop\Registration\Login\node_modules\@babel\runtime\helpers\regeneratorRuntime.js:86:17)
    at Generator._invoke (C:\Users\kanch\OneDrive\Desktop\Registration\Login\node_modules\@babel\runtime\helpers\regeneratorRuntime.js:66:24)
    at Generator.next (C:\Users\kanch\OneDrive\Desktop\Registration\Login\node_modules\@babel\runtime\helpers\regeneratorRuntime.js:117:21)
    at asyncGeneratorStep (C:\Users\kanch\OneDrive\Desktop\Registration\Login\node_modules\@babel\runtime\helpers\asyncToGenerator.js:3:24)
    at _next (C:\Users\kanch\OneDrive\Desktop\Registration\Login\node_modules\@babel\runtime\helpers\asyncToGenerator.js:25:9)
    at processTicksAndRejections (node:internal/process/task_queues:96:5) 
    url - /api/v1/users 
    method - GET 
    IP - ::1
   {"timestamp":"2022-07-18T07:13:21.676Z"}
error: 
    status - 500 
    message - TypeError: Cannot read properties of undefined (reading 'email')
    at _callee$ (C:\Users\kanch\OneDrive\Desktop\Registration\Login\src\services\/user.service.js:6:48)
    at tryCatch (C:\Users\kanch\OneDrive\Desktop\Registration\Login\node_modules\@babel\runtime\helpers\regeneratorRuntime.js:86:17)
    at Generator._invoke (C:\Users\kanch\OneDrive\Desktop\Registration\Login\node_modules\@babel\runtime\helpers\regeneratorRuntime.js:66:24)
    at Generator.next (C:\Users\kanch\OneDrive\Desktop\Registration\Login\node_modules\@babel\runtime\helpers\regeneratorRuntime.js:117:21)
    at asyncGeneratorStep (C:\Users\kanch\OneDrive\Desktop\Registration\Login\node_modules\@babel\runtime\helpers\asyncToGenerator.js:3:24)
    at _next (C:\Users\kanch\OneDrive\Desktop\Registration\Login\node_modules\@babel\runtime\helpers\asyncToGenerator.js:25:9)
    at C:\Users\kanch\OneDrive\Desktop\Registration\Login\node_modules\@babel\runtime\helpers\asyncToGenerator.js:32:7
    at new Promise (<anonymous>)
    at Object.<anonymous> (C:\Users\kanch\OneDrive\Desktop\Registration\Login\node_modules\@babel\runtime\helpers\asyncToGenerator.js:21:12)
    at Object.getAllUsers (C:\Users\kanch\OneDrive\Desktop\Registration\Login\src\services\/user.service.js:5:25)
    at _callee$ (C:\Users\kanch\OneDrive\Desktop\Registration\Login\src\controllers\/user.controller.js:12:36)
    at tryCatch (C:\Users\kanch\OneDrive\Desktop\Registration\Login\node_modules\@babel\runtime\helpers\regeneratorRuntime.js:86:17)
    at Generator._invoke (C:\Users\kanch\OneDrive\Desktop\Registration\Login\node_modules\@babel\runtime\helpers\regeneratorRuntime.js:66:24)
    at Generator.next (C:\Users\kanch\OneDrive\Desktop\Registration\Login\node_modules\@babel\runtime\helpers\regeneratorRuntime.js:117:21)
    at asyncGeneratorStep (C:\Users\kanch\OneDrive\Desktop\Registration\Login\node_modules\@babel\runtime\helpers\asyncToGenerator.js:3:24)
    at _next (C:\Users\kanch\OneDrive\Desktop\Registration\Login\node_modules\@babel\runtime\helpers\asyncToGenerator.js:25:9) 
    url - /api/v1/users 
    method - GET 
    IP - ::1
   {"timestamp":"2022-07-18T07:17:26.668Z"}
error: 
    status - 500 
    message - ReferenceError: userdata is not defined
    at _callee$ (C:\Users\kanch\OneDrive\Desktop\Registration\Login\src\services\/user.service.js:6:43)
    at tryCatch (C:\Users\kanch\OneDrive\Desktop\Registration\Login\node_modules\@babel\runtime\helpers\regeneratorRuntime.js:86:17)
    at Generator._invoke (C:\Users\kanch\OneDrive\Desktop\Registration\Login\node_modules\@babel\runtime\helpers\regeneratorRuntime.js:66:24)
    at Generator.next (C:\Users\kanch\OneDrive\Desktop\Registration\Login\node_modules\@babel\runtime\helpers\regeneratorRuntime.js:117:21)
    at asyncGeneratorStep (C:\Users\kanch\OneDrive\Desktop\Registration\Login\node_modules\@babel\runtime\helpers\asyncToGenerator.js:3:24)
    at _next (C:\Users\kanch\OneDrive\Desktop\Registration\Login\node_modules\@babel\runtime\helpers\asyncToGenerator.js:25:9)
    at C:\Users\kanch\OneDrive\Desktop\Registration\Login\node_modules\@babel\runtime\helpers\asyncToGenerator.js:32:7
    at new Promise (<anonymous>)
    at Object.<anonymous> (C:\Users\kanch\OneDrive\Desktop\Registration\Login\node_modules\@babel\runtime\helpers\asyncToGenerator.js:21:12)
    at Object.getAllUsers (C:\Users\kanch\OneDrive\Desktop\Registration\Login\src\services\/user.service.js:5:25)
    at _callee$ (C:\Users\kanch\OneDrive\Desktop\Registration\Login\src\controllers\/user.controller.js:12:36)
    at tryCatch (C:\Users\kanch\OneDrive\Desktop\Registration\Login\node_modules\@babel\runtime\helpers\regeneratorRuntime.js:86:17)
    at Generator._invoke (C:\Users\kanch\OneDrive\Desktop\Registration\Login\node_modules\@babel\runtime\helpers\regeneratorRuntime.js:66:24)
    at Generator.next (C:\Users\kanch\OneDrive\Desktop\Registration\Login\node_modules\@babel\runtime\helpers\regeneratorRuntime.js:117:21)
    at asyncGeneratorStep (C:\Users\kanch\OneDrive\Desktop\Registration\Login\node_modules\@babel\runtime\helpers\asyncToGenerator.js:3:24)
    at _next (C:\Users\kanch\OneDrive\Desktop\Registration\Login\node_modules\@babel\runtime\helpers\asyncToGenerator.js:25:9) 
    url - /api/v1/users 
    method - GET 
    IP - ::1
   {"timestamp":"2022-07-18T07:17:46.503Z"}
error: 
    status - 500 
    message - TypeError: Cannot read properties of undefined (reading '0')
    at _callee$ (C:\Users\kanch\OneDrive\Desktop\Registration\Login\src\services\/user.service.js:6:47)
    at tryCatch (C:\Users\kanch\OneDrive\Desktop\Registration\Login\node_modules\@babel\runtime\helpers\regeneratorRuntime.js:86:17)
    at Generator._invoke (C:\Users\kanch\OneDrive\Desktop\Registration\Login\node_modules\@babel\runtime\helpers\regeneratorRuntime.js:66:24)
    at Generator.next (C:\Users\kanch\OneDrive\Desktop\Registration\Login\node_modules\@babel\runtime\helpers\regeneratorRuntime.js:117:21)
    at asyncGeneratorStep (C:\Users\kanch\OneDrive\Desktop\Registration\Login\node_modules\@babel\runtime\helpers\asyncToGenerator.js:3:24)
    at _next (C:\Users\kanch\OneDrive\Desktop\Registration\Login\node_modules\@babel\runtime\helpers\asyncToGenerator.js:25:9)
    at C:\Users\kanch\OneDrive\Desktop\Registration\Login\node_modules\@babel\runtime\helpers\asyncToGenerator.js:32:7
    at new Promise (<anonymous>)
    at Object.<anonymous> (C:\Users\kanch\OneDrive\Desktop\Registration\Login\node_modules\@babel\runtime\helpers\asyncToGenerator.js:21:12)
    at Object.getAllUsers (C:\Users\kanch\OneDrive\Desktop\Registration\Login\src\services\/user.service.js:5:25)
    at _callee$ (C:\Users\kanch\OneDrive\Desktop\Registration\Login\src\controllers\/user.controller.js:12:36)
    at tryCatch (C:\Users\kanch\OneDrive\Desktop\Registration\Login\node_modules\@babel\runtime\helpers\regeneratorRuntime.js:86:17)
    at Generator._invoke (C:\Users\kanch\OneDrive\Desktop\Registration\Login\node_modules\@babel\runtime\helpers\regeneratorRuntime.js:66:24)
    at Generator.next (C:\Users\kanch\OneDrive\Desktop\Registration\Login\node_modules\@babel\runtime\helpers\regeneratorRuntime.js:117:21)
    at asyncGeneratorStep (C:\Users\kanch\OneDrive\Desktop\Registration\Login\node_modules\@babel\runtime\helpers\asyncToGenerator.js:3:24)
    at _next (C:\Users\kanch\OneDrive\Desktop\Registration\Login\node_modules\@babel\runtime\helpers\asyncToGenerator.js:25:9) 
    url - /api/v1/users 
    method - GET 
    IP - ::1
   {"timestamp":"2022-07-18T07:18:11.956Z"}
error: 
    status - 500 
    message - TypeError: Cannot read properties of undefined (reading '0')
    at _callee$ (C:\Users\kanch\OneDrive\Desktop\Registration\Login\src\services\/user.service.js:6:47)
    at tryCatch (C:\Users\kanch\OneDrive\Desktop\Registration\Login\node_modules\@babel\runtime\helpers\regeneratorRuntime.js:86:17)
    at Generator._invoke (C:\Users\kanch\OneDrive\Desktop\Registration\Login\node_modules\@babel\runtime\helpers\regeneratorRuntime.js:66:24)
    at Generator.next (C:\Users\kanch\OneDrive\Desktop\Registration\Login\node_modules\@babel\runtime\helpers\regeneratorRuntime.js:117:21)
    at asyncGeneratorStep (C:\Users\kanch\OneDrive\Desktop\Registration\Login\node_modules\@babel\runtime\helpers\asyncToGenerator.js:3:24)
    at _next (C:\Users\kanch\OneDrive\Desktop\Registration\Login\node_modules\@babel\runtime\helpers\asyncToGenerator.js:25:9)
    at C:\Users\kanch\OneDrive\Desktop\Registration\Login\node_modules\@babel\runtime\helpers\asyncToGenerator.js:32:7
    at new Promise (<anonymous>)
    at Object.<anonymous> (C:\Users\kanch\OneDrive\Desktop\Registration\Login\node_modules\@babel\runtime\helpers\asyncToGenerator.js:21:12)
    at Object.getAllUsers (C:\Users\kanch\OneDrive\Desktop\Registration\Login\src\services\/user.service.js:5:25)
    at _callee$ (C:\Users\kanch\OneDrive\Desktop\Registration\Login\src\controllers\/user.controller.js:12:36)
    at tryCatch (C:\Users\kanch\OneDrive\Desktop\Registration\Login\node_modules\@babel\runtime\helpers\regeneratorRuntime.js:86:17)
    at Generator._invoke (C:\Users\kanch\OneDrive\Desktop\Registration\Login\node_modules\@babel\runtime\helpers\regeneratorRuntime.js:66:24)
    at Generator.next (C:\Users\kanch\OneDrive\Desktop\Registration\Login\node_modules\@babel\runtime\helpers\regeneratorRuntime.js:117:21)
    at asyncGeneratorStep (C:\Users\kanch\OneDrive\Desktop\Registration\Login\node_modules\@babel\runtime\helpers\asyncToGenerator.js:3:24)
    at _next (C:\Users\kanch\OneDrive\Desktop\Registration\Login\node_modules\@babel\runtime\helpers\asyncToGenerator.js:25:9) 
    url - /api/v1/users 
    method - GET 
    IP - ::1
   {"timestamp":"2022-07-18T07:18:16.208Z"}
error: 
    status - 500 
    message - ReferenceError: schema is not defined
    at _callee$ (C:\Users\kanch\OneDrive\Desktop\Registration\Login\src\services\/user.service.js:8:43)
    at tryCatch (C:\Users\kanch\OneDrive\Desktop\Registration\Login\node_modules\@babel\runtime\helpers\regeneratorRuntime.js:86:17)
    at Generator._invoke (C:\Users\kanch\OneDrive\Desktop\Registration\Login\node_modules\@babel\runtime\helpers\regeneratorRuntime.js:66:24)
    at Generator.next (C:\Users\kanch\OneDrive\Desktop\Registration\Login\node_modules\@babel\runtime\helpers\regeneratorRuntime.js:117:21)
    at asyncGeneratorStep (C:\Users\kanch\OneDrive\Desktop\Registration\Login\node_modules\@babel\runtime\helpers\asyncToGenerator.js:3:24)
    at _next (C:\Users\kanch\OneDrive\Desktop\Registration\Login\node_modules\@babel\runtime\helpers\asyncToGenerator.js:25:9)
    at C:\Users\kanch\OneDrive\Desktop\Registration\Login\node_modules\@babel\runtime\helpers\asyncToGenerator.js:32:7
    at new Promise (<anonymous>)
    at Object.<anonymous> (C:\Users\kanch\OneDrive\Desktop\Registration\Login\node_modules\@babel\runtime\helpers\asyncToGenerator.js:21:12)
    at Object.getAllUsers (C:\Users\kanch\OneDrive\Desktop\Registration\Login\src\services\/user.service.js:7:25)
    at _callee$ (C:\Users\kanch\OneDrive\Desktop\Registration\Login\src\controllers\/user.controller.js:12:36)
    at tryCatch (C:\Users\kanch\OneDrive\Desktop\Registration\Login\node_modules\@babel\runtime\helpers\regeneratorRuntime.js:86:17)
    at Generator._invoke (C:\Users\kanch\OneDrive\Desktop\Registration\Login\node_modules\@babel\runtime\helpers\regeneratorRuntime.js:66:24)
    at Generator.next (C:\Users\kanch\OneDrive\Desktop\Registration\Login\node_modules\@babel\runtime\helpers\regeneratorRuntime.js:117:21)
    at asyncGeneratorStep (C:\Users\kanch\OneDrive\Desktop\Registration\Login\node_modules\@babel\runtime\helpers\asyncToGenerator.js:3:24)
    at _next (C:\Users\kanch\OneDrive\Desktop\Registration\Login\node_modules\@babel\runtime\helpers\asyncToGenerator.js:25:9) 
    url - /api/v1/users 
    method - GET 
    IP - ::1
   {"timestamp":"2022-07-18T07:21:43.159Z"}
error: 
    status - 500 
    message - TypeError: Cannot read properties of undefined (reading 'email')
    at _callee$ (C:\Users\kanch\OneDrive\Desktop\Registration\Login\src\services\/user.service.js:12:37)
    at tryCatch (C:\Users\kanch\OneDrive\Desktop\Registration\Login\node_modules\@babel\runtime\helpers\regeneratorRuntime.js:86:17)
    at Generator._invoke (C:\Users\kanch\OneDrive\Desktop\Registration\Login\node_modules\@babel\runtime\helpers\regeneratorRuntime.js:66:24)
    at Generator.next (C:\Users\kanch\OneDrive\Desktop\Registration\Login\node_modules\@babel\runtime\helpers\regeneratorRuntime.js:117:21)
    at asyncGeneratorStep (C:\Users\kanch\OneDrive\Desktop\Registration\Login\node_modules\@babel\runtime\helpers\asyncToGenerator.js:3:24)
    at _next (C:\Users\kanch\OneDrive\Desktop\Registration\Login\node_modules\@babel\runtime\helpers\asyncToGenerator.js:25:9)
    at processTicksAndRejections (node:internal/process/task_queues:96:5) 
    url - /api/v1/users 
    method - GET 
    IP - ::1
   {"timestamp":"2022-07-18T07:24:38.206Z"}
error: 
    status - 500 
    message - ReferenceError: email is not defined
    at _callee$ (C:\Users\kanch\OneDrive\Desktop\Registration\Login\src\services\/user.service.js:12:23)
    at tryCatch (C:\Users\kanch\OneDrive\Desktop\Registration\Login\node_modules\@babel\runtime\helpers\regeneratorRuntime.js:86:17)
    at Generator._invoke (C:\Users\kanch\OneDrive\Desktop\Registration\Login\node_modules\@babel\runtime\helpers\regeneratorRuntime.js:66:24)
    at Generator.next (C:\Users\kanch\OneDrive\Desktop\Registration\Login\node_modules\@babel\runtime\helpers\regeneratorRuntime.js:117:21)
    at asyncGeneratorStep (C:\Users\kanch\OneDrive\Desktop\Registration\Login\node_modules\@babel\runtime\helpers\asyncToGenerator.js:3:24)
    at _next (C:\Users\kanch\OneDrive\Desktop\Registration\Login\node_modules\@babel\runtime\helpers\asyncToGenerator.js:25:9)
    at processTicksAndRejections (node:internal/process/task_queues:96:5) 
    url - /api/v1/users 
    method - GET 
    IP - ::1
   {"timestamp":"2022-07-18T07:25:09.930Z"}
error: 
    status - 500 
    message - ReferenceError: email is not defined
    at _callee$ (C:\Users\kanch\OneDrive\Desktop\Registration\Login\src\services\/user.service.js:12:23)
    at tryCatch (C:\Users\kanch\OneDrive\Desktop\Registration\Login\node_modules\@babel\runtime\helpers\regeneratorRuntime.js:86:17)
    at Generator._invoke (C:\Users\kanch\OneDrive\Desktop\Registration\Login\node_modules\@babel\runtime\helpers\regeneratorRuntime.js:66:24)
    at Generator.next (C:\Users\kanch\OneDrive\Desktop\Registration\Login\node_modules\@babel\runtime\helpers\regeneratorRuntime.js:117:21)
    at asyncGeneratorStep (C:\Users\kanch\OneDrive\Desktop\Registration\Login\node_modules\@babel\runtime\helpers\asyncToGenerator.js:3:24)
    at _next (C:\Users\kanch\OneDrive\Desktop\Registration\Login\node_modules\@babel\runtime\helpers\asyncToGenerator.js:25:9)
    at processTicksAndRejections (node:internal/process/task_queues:96:5) 
    url - /api/v1/users 
    method - GET 
    IP - ::1
   {"timestamp":"2022-07-18T07:25:23.964Z"}
error: 
    status - 500 
    message - ReferenceError: model is not defined
    at _callee$ (C:\Users\kanch\OneDrive\Desktop\FundooNotes\Login\src\services\/user.service.js:8:24)
    at tryCatch (C:\Users\kanch\OneDrive\Desktop\FundooNotes\Login\node_modules\@babel\runtime\helpers\regeneratorRuntime.js:86:17)
    at Generator._invoke (C:\Users\kanch\OneDrive\Desktop\FundooNotes\Login\node_modules\@babel\runtime\helpers\regeneratorRuntime.js:66:24)
    at Generator.next (C:\Users\kanch\OneDrive\Desktop\FundooNotes\Login\node_modules\@babel\runtime\helpers\regeneratorRuntime.js:117:21)
    at asyncGeneratorStep (C:\Users\kanch\OneDrive\Desktop\FundooNotes\Login\node_modules\@babel\runtime\helpers\asyncToGenerator.js:3:24)
    at _next (C:\Users\kanch\OneDrive\Desktop\FundooNotes\Login\node_modules\@babel\runtime\helpers\asyncToGenerator.js:25:9)
    at processTicksAndRejections (node:internal/process/task_queues:96:5) 
    url - /api/v1/users 
    method - GET 
    IP - ::1
   {"timestamp":"2022-07-18T08:51:41.091Z"}
error: 
    status - 500 
    message - TypeError: Cannot read properties of undefined (reading 'email')
    at _callee$ (C:\Users\kanch\OneDrive\Desktop\FundooNotes\Login\src\services\/user.service.js:10:37)
    at tryCatch (C:\Users\kanch\OneDrive\Desktop\FundooNotes\Login\node_modules\@babel\runtime\helpers\regeneratorRuntime.js:86:17)
    at Generator._invoke (C:\Users\kanch\OneDrive\Desktop\FundooNotes\Login\node_modules\@babel\runtime\helpers\regeneratorRuntime.js:66:24)
    at Generator.next (C:\Users\kanch\OneDrive\Desktop\FundooNotes\Login\node_modules\@babel\runtime\helpers\regeneratorRuntime.js:117:21)
    at asyncGeneratorStep (C:\Users\kanch\OneDrive\Desktop\FundooNotes\Login\node_modules\@babel\runtime\helpers\asyncToGenerator.js:3:24)
    at _next (C:\Users\kanch\OneDrive\Desktop\FundooNotes\Login\node_modules\@babel\runtime\helpers\asyncToGenerator.js:25:9)
    at processTicksAndRejections (node:internal/process/task_queues:96:5) 
    url - /api/v1/users 
    method - GET 
    IP - ::1
   {"timestamp":"2022-07-18T08:59:14.335Z"}
error: 
    status - 500 
    message - ReferenceError: User is not defined
    at _callee$ (C:\Users\kanch\OneDrive\Desktop\FundooNotes\Login\src\services\/user.service.js:4:28)
    at tryCatch (C:\Users\kanch\OneDrive\Desktop\FundooNotes\Login\node_modules\@babel\runtime\helpers\regeneratorRuntime.js:86:17)
    at Generator._invoke (C:\Users\kanch\OneDrive\Desktop\FundooNotes\Login\node_modules\@babel\runtime\helpers\regeneratorRuntime.js:66:24)
    at Generator.next (C:\Users\kanch\OneDrive\Desktop\FundooNotes\Login\node_modules\@babel\runtime\helpers\regeneratorRuntime.js:117:21)
    at asyncGeneratorStep (C:\Users\kanch\OneDrive\Desktop\FundooNotes\Login\node_modules\@babel\runtime\helpers\asyncToGenerator.js:3:24)
    at _next (C:\Users\kanch\OneDrive\Desktop\FundooNotes\Login\node_modules\@babel\runtime\helpers\asyncToGenerator.js:25:9)
    at C:\Users\kanch\OneDrive\Desktop\FundooNotes\Login\node_modules\@babel\runtime\helpers\asyncToGenerator.js:32:7
    at new Promise (<anonymous>)
    at Object.<anonymous> (C:\Users\kanch\OneDrive\Desktop\FundooNotes\Login\node_modules\@babel\runtime\helpers\asyncToGenerator.js:21:12)
    at Object.getAllUsers (C:\Users\kanch\OneDrive\Desktop\FundooNotes\Login\src\services\/user.service.js:4:25)
    at _callee$ (C:\Users\kanch\OneDrive\Desktop\FundooNotes\Login\src\controllers\/user.controller.js:12:36)
    at tryCatch (C:\Users\kanch\OneDrive\Desktop\FundooNotes\Login\node_modules\@babel\runtime\helpers\regeneratorRuntime.js:86:17)
    at Generator._invoke (C:\Users\kanch\OneDrive\Desktop\FundooNotes\Login\node_modules\@babel\runtime\helpers\regeneratorRuntime.js:66:24)
    at Generator.next (C:\Users\kanch\OneDrive\Desktop\FundooNotes\Login\node_modules\@babel\runtime\helpers\regeneratorRuntime.js:117:21)
    at asyncGeneratorStep (C:\Users\kanch\OneDrive\Desktop\FundooNotes\Login\node_modules\@babel\runtime\helpers\asyncToGenerator.js:3:24)
    at _next (C:\Users\kanch\OneDrive\Desktop\FundooNotes\Login\node_modules\@babel\runtime\helpers\asyncToGenerator.js:25:9) 
    url - /api/v1/users 
    method - GET 
    IP - ::1
   {"timestamp":"2022-07-18T09:00:59.738Z"}
error: 
    status - 500 
    message - ReferenceError: User is not defined
    at _callee$ (C:\Users\kanch\OneDrive\Desktop\FundooNotes\Login\src\services\/user.service.js:5:28)
    at tryCatch (C:\Users\kanch\OneDrive\Desktop\FundooNotes\Login\node_modules\@babel\runtime\helpers\regeneratorRuntime.js:86:17)
    at Generator._invoke (C:\Users\kanch\OneDrive\Desktop\FundooNotes\Login\node_modules\@babel\runtime\helpers\regeneratorRuntime.js:66:24)
    at Generator.next (C:\Users\kanch\OneDrive\Desktop\FundooNotes\Login\node_modules\@babel\runtime\helpers\regeneratorRuntime.js:117:21)
    at asyncGeneratorStep (C:\Users\kanch\OneDrive\Desktop\FundooNotes\Login\node_modules\@babel\runtime\helpers\asyncToGenerator.js:3:24)
    at _next (C:\Users\kanch\OneDrive\Desktop\FundooNotes\Login\node_modules\@babel\runtime\helpers\asyncToGenerator.js:25:9)
    at C:\Users\kanch\OneDrive\Desktop\FundooNotes\Login\node_modules\@babel\runtime\helpers\asyncToGenerator.js:32:7
    at new Promise (<anonymous>)
    at Object.<anonymous> (C:\Users\kanch\OneDrive\Desktop\FundooNotes\Login\node_modules\@babel\runtime\helpers\asyncToGenerator.js:21:12)
    at Object.getAllUsers (C:\Users\kanch\OneDrive\Desktop\FundooNotes\Login\src\services\/user.service.js:5:25)
    at _callee$ (C:\Users\kanch\OneDrive\Desktop\FundooNotes\Login\src\controllers\/user.controller.js:12:36)
    at tryCatch (C:\Users\kanch\OneDrive\Desktop\FundooNotes\Login\node_modules\@babel\runtime\helpers\regeneratorRuntime.js:86:17)
    at Generator._invoke (C:\Users\kanch\OneDrive\Desktop\FundooNotes\Login\node_modules\@babel\runtime\helpers\regeneratorRuntime.js:66:24)
    at Generator.next (C:\Users\kanch\OneDrive\Desktop\FundooNotes\Login\node_modules\@babel\runtime\helpers\regeneratorRuntime.js:117:21)
    at asyncGeneratorStep (C:\Users\kanch\OneDrive\Desktop\FundooNotes\Login\node_modules\@babel\runtime\helpers\asyncToGenerator.js:3:24)
    at _next (C:\Users\kanch\OneDrive\Desktop\FundooNotes\Login\node_modules\@babel\runtime\helpers\asyncToGenerator.js:25:9) 
    url - /api/v1/users 
    method - GET 
    IP - ::1
   {"timestamp":"2022-07-18T09:02:01.718Z"}
error: 
    status - 500 
    message - ReferenceError: User is not defined
    at _callee$ (C:\Users\kanch\OneDrive\Desktop\FundooNotes\Login\src\services\/user.service.js:5:28)
    at tryCatch (C:\Users\kanch\OneDrive\Desktop\FundooNotes\Login\node_modules\@babel\runtime\helpers\regeneratorRuntime.js:86:17)
    at Generator._invoke (C:\Users\kanch\OneDrive\Desktop\FundooNotes\Login\node_modules\@babel\runtime\helpers\regeneratorRuntime.js:66:24)
    at Generator.next (C:\Users\kanch\OneDrive\Desktop\FundooNotes\Login\node_modules\@babel\runtime\helpers\regeneratorRuntime.js:117:21)
    at asyncGeneratorStep (C:\Users\kanch\OneDrive\Desktop\FundooNotes\Login\node_modules\@babel\runtime\helpers\asyncToGenerator.js:3:24)
    at _next (C:\Users\kanch\OneDrive\Desktop\FundooNotes\Login\node_modules\@babel\runtime\helpers\asyncToGenerator.js:25:9)
    at C:\Users\kanch\OneDrive\Desktop\FundooNotes\Login\node_modules\@babel\runtime\helpers\asyncToGenerator.js:32:7
    at new Promise (<anonymous>)
    at Object.<anonymous> (C:\Users\kanch\OneDrive\Desktop\FundooNotes\Login\node_modules\@babel\runtime\helpers\asyncToGenerator.js:21:12)
    at Object.getAllUsers (C:\Users\kanch\OneDrive\Desktop\FundooNotes\Login\src\services\/user.service.js:5:25)
    at _callee$ (C:\Users\kanch\OneDrive\Desktop\FundooNotes\Login\src\controllers\/user.controller.js:12:36)
    at tryCatch (C:\Users\kanch\OneDrive\Desktop\FundooNotes\Login\node_modules\@babel\runtime\helpers\regeneratorRuntime.js:86:17)
    at Generator._invoke (C:\Users\kanch\OneDrive\Desktop\FundooNotes\Login\node_modules\@babel\runtime\helpers\regeneratorRuntime.js:66:24)
    at Generator.next (C:\Users\kanch\OneDrive\Desktop\FundooNotes\Login\node_modules\@babel\runtime\helpers\regeneratorRuntime.js:117:21)
    at asyncGeneratorStep (C:\Users\kanch\OneDrive\Desktop\FundooNotes\Login\node_modules\@babel\runtime\helpers\asyncToGenerator.js:3:24)
    at _next (C:\Users\kanch\OneDrive\Desktop\FundooNotes\Login\node_modules\@babel\runtime\helpers\asyncToGenerator.js:25:9) 
    url - /api/v1/users 
    method - GET 
    IP - ::1
   {"timestamp":"2022-07-18T09:06:01.776Z"}
error: 
    status - 500 
    message - ReferenceError: User is not defined
    at _callee$ (C:\Users\kanch\OneDrive\Desktop\FundooNotes\Login\src\services\/user.service.js:4:28)
    at tryCatch (C:\Users\kanch\OneDrive\Desktop\FundooNotes\Login\node_modules\@babel\runtime\helpers\regeneratorRuntime.js:86:17)
    at Generator._invoke (C:\Users\kanch\OneDrive\Desktop\FundooNotes\Login\node_modules\@babel\runtime\helpers\regeneratorRuntime.js:66:24)
    at Generator.next (C:\Users\kanch\OneDrive\Desktop\FundooNotes\Login\node_modules\@babel\runtime\helpers\regeneratorRuntime.js:117:21)
    at asyncGeneratorStep (C:\Users\kanch\OneDrive\Desktop\FundooNotes\Login\node_modules\@babel\runtime\helpers\asyncToGenerator.js:3:24)
    at _next (C:\Users\kanch\OneDrive\Desktop\FundooNotes\Login\node_modules\@babel\runtime\helpers\asyncToGenerator.js:25:9)
    at C:\Users\kanch\OneDrive\Desktop\FundooNotes\Login\node_modules\@babel\runtime\helpers\asyncToGenerator.js:32:7
    at new Promise (<anonymous>)
    at Object.<anonymous> (C:\Users\kanch\OneDrive\Desktop\FundooNotes\Login\node_modules\@babel\runtime\helpers\asyncToGenerator.js:21:12)
    at Object.getAllUsers (C:\Users\kanch\OneDrive\Desktop\FundooNotes\Login\src\services\/user.service.js:4:25)
    at _callee$ (C:\Users\kanch\OneDrive\Desktop\FundooNotes\Login\src\controllers\/user.controller.js:12:36)
    at tryCatch (C:\Users\kanch\OneDrive\Desktop\FundooNotes\Login\node_modules\@babel\runtime\helpers\regeneratorRuntime.js:86:17)
    at Generator._invoke (C:\Users\kanch\OneDrive\Desktop\FundooNotes\Login\node_modules\@babel\runtime\helpers\regeneratorRuntime.js:66:24)
    at Generator.next (C:\Users\kanch\OneDrive\Desktop\FundooNotes\Login\node_modules\@babel\runtime\helpers\regeneratorRuntime.js:117:21)
    at asyncGeneratorStep (C:\Users\kanch\OneDrive\Desktop\FundooNotes\Login\node_modules\@babel\runtime\helpers\asyncToGenerator.js:3:24)
    at _next (C:\Users\kanch\OneDrive\Desktop\FundooNotes\Login\node_modules\@babel\runtime\helpers\asyncToGenerator.js:25:9) 
    url - /api/v1/users 
    method - GET 
    IP - ::1
   {"timestamp":"2022-07-18T09:06:30.819Z"}
error: 
    status - 500 
    message - ReferenceError: User is not defined
    at _callee$ (C:\Users\kanch\OneDrive\Desktop\FundooNotes\Login\src\services\/user.service.js:4:28)
    at tryCatch (C:\Users\kanch\OneDrive\Desktop\FundooNotes\Login\node_modules\@babel\runtime\helpers\regeneratorRuntime.js:86:17)
    at Generator._invoke (C:\Users\kanch\OneDrive\Desktop\FundooNotes\Login\node_modules\@babel\runtime\helpers\regeneratorRuntime.js:66:24)
    at Generator.next (C:\Users\kanch\OneDrive\Desktop\FundooNotes\Login\node_modules\@babel\runtime\helpers\regeneratorRuntime.js:117:21)
    at asyncGeneratorStep (C:\Users\kanch\OneDrive\Desktop\FundooNotes\Login\node_modules\@babel\runtime\helpers\asyncToGenerator.js:3:24)
    at _next (C:\Users\kanch\OneDrive\Desktop\FundooNotes\Login\node_modules\@babel\runtime\helpers\asyncToGenerator.js:25:9)
    at C:\Users\kanch\OneDrive\Desktop\FundooNotes\Login\node_modules\@babel\runtime\helpers\asyncToGenerator.js:32:7
    at new Promise (<anonymous>)
    at Object.<anonymous> (C:\Users\kanch\OneDrive\Desktop\FundooNotes\Login\node_modules\@babel\runtime\helpers\asyncToGenerator.js:21:12)
    at Object.getAllUsers (C:\Users\kanch\OneDrive\Desktop\FundooNotes\Login\src\services\/user.service.js:4:25)
    at _callee$ (C:\Users\kanch\OneDrive\Desktop\FundooNotes\Login\src\controllers\/user.controller.js:12:36)
    at tryCatch (C:\Users\kanch\OneDrive\Desktop\FundooNotes\Login\node_modules\@babel\runtime\helpers\regeneratorRuntime.js:86:17)
    at Generator._invoke (C:\Users\kanch\OneDrive\Desktop\FundooNotes\Login\node_modules\@babel\runtime\helpers\regeneratorRuntime.js:66:24)
    at Generator.next (C:\Users\kanch\OneDrive\Desktop\FundooNotes\Login\node_modules\@babel\runtime\helpers\regeneratorRuntime.js:117:21)
    at asyncGeneratorStep (C:\Users\kanch\OneDrive\Desktop\FundooNotes\Login\node_modules\@babel\runtime\helpers\asyncToGenerator.js:3:24)
    at _next (C:\Users\kanch\OneDrive\Desktop\FundooNotes\Login\node_modules\@babel\runtime\helpers\asyncToGenerator.js:25:9) 
    url - /api/v1/users 
    method - GET 
    IP - ::1
   {"timestamp":"2022-07-18T09:10:18.776Z"}
error: 
    status - 500 
    message - ReferenceError: User is not defined
    at _callee$ (C:\Users\kanch\OneDrive\Desktop\FundooNotes\Login\src\services\/user.service.js:3:28)
    at tryCatch (C:\Users\kanch\OneDrive\Desktop\FundooNotes\Login\node_modules\@babel\runtime\helpers\regeneratorRuntime.js:86:17)
    at Generator._invoke (C:\Users\kanch\OneDrive\Desktop\FundooNotes\Login\node_modules\@babel\runtime\helpers\regeneratorRuntime.js:66:24)
    at Generator.next (C:\Users\kanch\OneDrive\Desktop\FundooNotes\Login\node_modules\@babel\runtime\helpers\regeneratorRuntime.js:117:21)
    at asyncGeneratorStep (C:\Users\kanch\OneDrive\Desktop\FundooNotes\Login\node_modules\@babel\runtime\helpers\asyncToGenerator.js:3:24)
    at _next (C:\Users\kanch\OneDrive\Desktop\FundooNotes\Login\node_modules\@babel\runtime\helpers\asyncToGenerator.js:25:9)
    at C:\Users\kanch\OneDrive\Desktop\FundooNotes\Login\node_modules\@babel\runtime\helpers\asyncToGenerator.js:32:7
    at new Promise (<anonymous>)
    at Object.<anonymous> (C:\Users\kanch\OneDrive\Desktop\FundooNotes\Login\node_modules\@babel\runtime\helpers\asyncToGenerator.js:21:12)
    at Object.getAllUsers (C:\Users\kanch\OneDrive\Desktop\FundooNotes\Login\src\services\/user.service.js:3:25)
    at _callee$ (C:\Users\kanch\OneDrive\Desktop\FundooNotes\Login\src\controllers\/user.controller.js:12:36)
    at tryCatch (C:\Users\kanch\OneDrive\Desktop\FundooNotes\Login\node_modules\@babel\runtime\helpers\regeneratorRuntime.js:86:17)
    at Generator._invoke (C:\Users\kanch\OneDrive\Desktop\FundooNotes\Login\node_modules\@babel\runtime\helpers\regeneratorRuntime.js:66:24)
    at Generator.next (C:\Users\kanch\OneDrive\Desktop\FundooNotes\Login\node_modules\@babel\runtime\helpers\regeneratorRuntime.js:117:21)
    at asyncGeneratorStep (C:\Users\kanch\OneDrive\Desktop\FundooNotes\Login\node_modules\@babel\runtime\helpers\asyncToGenerator.js:3:24)
    at _next (C:\Users\kanch\OneDrive\Desktop\FundooNotes\Login\node_modules\@babel\runtime\helpers\asyncToGenerator.js:25:9) 
    url - /api/v1/users 
    method - GET 
    IP - ::1
   {"timestamp":"2022-07-18T09:13:11.996Z"}
error: 
    status - 500 
    message - ReferenceError: User is not defined
    at _callee$ (C:\Users\kanch\OneDrive\Desktop\FundooNotes\Login\src\services\/user.service.js:3:28)
    at tryCatch (C:\Users\kanch\OneDrive\Desktop\FundooNotes\Login\node_modules\@babel\runtime\helpers\regeneratorRuntime.js:86:17)
    at Generator._invoke (C:\Users\kanch\OneDrive\Desktop\FundooNotes\Login\node_modules\@babel\runtime\helpers\regeneratorRuntime.js:66:24)
    at Generator.next (C:\Users\kanch\OneDrive\Desktop\FundooNotes\Login\node_modules\@babel\runtime\helpers\regeneratorRuntime.js:117:21)
    at asyncGeneratorStep (C:\Users\kanch\OneDrive\Desktop\FundooNotes\Login\node_modules\@babel\runtime\helpers\asyncToGenerator.js:3:24)
    at _next (C:\Users\kanch\OneDrive\Desktop\FundooNotes\Login\node_modules\@babel\runtime\helpers\asyncToGenerator.js:25:9)
    at C:\Users\kanch\OneDrive\Desktop\FundooNotes\Login\node_modules\@babel\runtime\helpers\asyncToGenerator.js:32:7
    at new Promise (<anonymous>)
    at Object.<anonymous> (C:\Users\kanch\OneDrive\Desktop\FundooNotes\Login\node_modules\@babel\runtime\helpers\asyncToGenerator.js:21:12)
    at Object.getAllUsers (C:\Users\kanch\OneDrive\Desktop\FundooNotes\Login\src\services\/user.service.js:3:25)
    at _callee$ (C:\Users\kanch\OneDrive\Desktop\FundooNotes\Login\src\controllers\/user.controller.js:12:36)
    at tryCatch (C:\Users\kanch\OneDrive\Desktop\FundooNotes\Login\node_modules\@babel\runtime\helpers\regeneratorRuntime.js:86:17)
    at Generator._invoke (C:\Users\kanch\OneDrive\Desktop\FundooNotes\Login\node_modules\@babel\runtime\helpers\regeneratorRuntime.js:66:24)
    at Generator.next (C:\Users\kanch\OneDrive\Desktop\FundooNotes\Login\node_modules\@babel\runtime\helpers\regeneratorRuntime.js:117:21)
    at asyncGeneratorStep (C:\Users\kanch\OneDrive\Desktop\FundooNotes\Login\node_modules\@babel\runtime\helpers\asyncToGenerator.js:3:24)
    at _next (C:\Users\kanch\OneDrive\Desktop\FundooNotes\Login\node_modules\@babel\runtime\helpers\asyncToGenerator.js:25:9) 
    url - /api/v1/users 
    method - GET 
    IP - ::1
   {"timestamp":"2022-07-18T09:13:41.878Z"}
error: 
    status - 500 
    message - ReferenceError: User is not defined
    at _callee$ (C:\Users\kanch\OneDrive\Desktop\FundooNotes\Login\src\services\/user.service.js:3:28)
    at tryCatch (C:\Users\kanch\OneDrive\Desktop\FundooNotes\Login\node_modules\@babel\runtime\helpers\regeneratorRuntime.js:86:17)
    at Generator._invoke (C:\Users\kanch\OneDrive\Desktop\FundooNotes\Login\node_modules\@babel\runtime\helpers\regeneratorRuntime.js:66:24)
    at Generator.next (C:\Users\kanch\OneDrive\Desktop\FundooNotes\Login\node_modules\@babel\runtime\helpers\regeneratorRuntime.js:117:21)
    at asyncGeneratorStep (C:\Users\kanch\OneDrive\Desktop\FundooNotes\Login\node_modules\@babel\runtime\helpers\asyncToGenerator.js:3:24)
    at _next (C:\Users\kanch\OneDrive\Desktop\FundooNotes\Login\node_modules\@babel\runtime\helpers\asyncToGenerator.js:25:9)
    at C:\Users\kanch\OneDrive\Desktop\FundooNotes\Login\node_modules\@babel\runtime\helpers\asyncToGenerator.js:32:7
    at new Promise (<anonymous>)
    at Object.<anonymous> (C:\Users\kanch\OneDrive\Desktop\FundooNotes\Login\node_modules\@babel\runtime\helpers\asyncToGenerator.js:21:12)
    at Object.getAllUsers (C:\Users\kanch\OneDrive\Desktop\FundooNotes\Login\src\services\/user.service.js:3:25)
    at _callee$ (C:\Users\kanch\OneDrive\Desktop\FundooNotes\Login\src\controllers\/user.controller.js:12:36)
    at tryCatch (C:\Users\kanch\OneDrive\Desktop\FundooNotes\Login\node_modules\@babel\runtime\helpers\regeneratorRuntime.js:86:17)
    at Generator._invoke (C:\Users\kanch\OneDrive\Desktop\FundooNotes\Login\node_modules\@babel\runtime\helpers\regeneratorRuntime.js:66:24)
    at Generator.next (C:\Users\kanch\OneDrive\Desktop\FundooNotes\Login\node_modules\@babel\runtime\helpers\regeneratorRuntime.js:117:21)
    at asyncGeneratorStep (C:\Users\kanch\OneDrive\Desktop\FundooNotes\Login\node_modules\@babel\runtime\helpers\asyncToGenerator.js:3:24)
    at _next (C:\Users\kanch\OneDrive\Desktop\FundooNotes\Login\node_modules\@babel\runtime\helpers\asyncToGenerator.js:25:9) 
    url - /api/v1/users 
    method - GET 
    IP - ::1
   {"timestamp":"2022-07-18T09:17:02.119Z"}
error: 
    status - 500 
    message - TypeError: Cannot read properties of undefined (reading 'body')
    at _callee$ (C:\Users\kanch\OneDrive\Desktop\FundooNotes\Login\src\services\/user.service.js:5:45)
    at tryCatch (C:\Users\kanch\OneDrive\Desktop\FundooNotes\Login\node_modules\@babel\runtime\helpers\regeneratorRuntime.js:86:17)
    at Generator._invoke (C:\Users\kanch\OneDrive\Desktop\FundooNotes\Login\node_modules\@babel\runtime\helpers\regeneratorRuntime.js:66:24)
    at Generator.next (C:\Users\kanch\OneDrive\Desktop\FundooNotes\Login\node_modules\@babel\runtime\helpers\regeneratorRuntime.js:117:21)
    at asyncGeneratorStep (C:\Users\kanch\OneDrive\Desktop\FundooNotes\Login\node_modules\@babel\runtime\helpers\asyncToGenerator.js:3:24)
    at _next (C:\Users\kanch\OneDrive\Desktop\FundooNotes\Login\node_modules\@babel\runtime\helpers\asyncToGenerator.js:25:9)
    at C:\Users\kanch\OneDrive\Desktop\FundooNotes\Login\node_modules\@babel\runtime\helpers\asyncToGenerator.js:32:7
    at new Promise (<anonymous>)
    at Object.<anonymous> (C:\Users\kanch\OneDrive\Desktop\FundooNotes\Login\node_modules\@babel\runtime\helpers\asyncToGenerator.js:21:12)
    at Object.getAllUsers (C:\Users\kanch\OneDrive\Desktop\FundooNotes\Login\src\services\/user.service.js:4:25)
    at _callee$ (C:\Users\kanch\OneDrive\Desktop\FundooNotes\Login\src\controllers\/user.controller.js:12:36)
    at tryCatch (C:\Users\kanch\OneDrive\Desktop\FundooNotes\Login\node_modules\@babel\runtime\helpers\regeneratorRuntime.js:86:17)
    at Generator._invoke (C:\Users\kanch\OneDrive\Desktop\FundooNotes\Login\node_modules\@babel\runtime\helpers\regeneratorRuntime.js:66:24)
    at Generator.next (C:\Users\kanch\OneDrive\Desktop\FundooNotes\Login\node_modules\@babel\runtime\helpers\regeneratorRuntime.js:117:21)
    at asyncGeneratorStep (C:\Users\kanch\OneDrive\Desktop\FundooNotes\Login\node_modules\@babel\runtime\helpers\asyncToGenerator.js:3:24)
    at _next (C:\Users\kanch\OneDrive\Desktop\FundooNotes\Login\node_modules\@babel\runtime\helpers\asyncToGenerator.js:25:9) 
    url - /api/v1/users 
    method - GET 
    IP - ::1
   {"timestamp":"2022-07-18T09:30:36.506Z"}
error: 
    status - 500 
    message - TypeError: Cannot read properties of undefined (reading 'email')
    at _callee$ (C:\Users\kanch\OneDrive\Desktop\FundooNotes\Login\src\services\/user.service.js:5:45)
    at tryCatch (C:\Users\kanch\OneDrive\Desktop\FundooNotes\Login\node_modules\@babel\runtime\helpers\regeneratorRuntime.js:86:17)
    at Generator._invoke (C:\Users\kanch\OneDrive\Desktop\FundooNotes\Login\node_modules\@babel\runtime\helpers\regeneratorRuntime.js:66:24)
    at Generator.next (C:\Users\kanch\OneDrive\Desktop\FundooNotes\Login\node_modules\@babel\runtime\helpers\regeneratorRuntime.js:117:21)
    at asyncGeneratorStep (C:\Users\kanch\OneDrive\Desktop\FundooNotes\Login\node_modules\@babel\runtime\helpers\asyncToGenerator.js:3:24)
    at _next (C:\Users\kanch\OneDrive\Desktop\FundooNotes\Login\node_modules\@babel\runtime\helpers\asyncToGenerator.js:25:9)
    at C:\Users\kanch\OneDrive\Desktop\FundooNotes\Login\node_modules\@babel\runtime\helpers\asyncToGenerator.js:32:7
    at new Promise (<anonymous>)
    at Object.<anonymous> (C:\Users\kanch\OneDrive\Desktop\FundooNotes\Login\node_modules\@babel\runtime\helpers\asyncToGenerator.js:21:12)
    at Object.getAllUsers (C:\Users\kanch\OneDrive\Desktop\FundooNotes\Login\src\services\/user.service.js:4:25)
    at _callee$ (C:\Users\kanch\OneDrive\Desktop\FundooNotes\Login\src\controllers\/user.controller.js:12:36)
    at tryCatch (C:\Users\kanch\OneDrive\Desktop\FundooNotes\Login\node_modules\@babel\runtime\helpers\regeneratorRuntime.js:86:17)
    at Generator._invoke (C:\Users\kanch\OneDrive\Desktop\FundooNotes\Login\node_modules\@babel\runtime\helpers\regeneratorRuntime.js:66:24)
    at Generator.next (C:\Users\kanch\OneDrive\Desktop\FundooNotes\Login\node_modules\@babel\runtime\helpers\regeneratorRuntime.js:117:21)
    at asyncGeneratorStep (C:\Users\kanch\OneDrive\Desktop\FundooNotes\Login\node_modules\@babel\runtime\helpers\asyncToGenerator.js:3:24)
    at _next (C:\Users\kanch\OneDrive\Desktop\FundooNotes\Login\node_modules\@babel\runtime\helpers\asyncToGenerator.js:25:9) 
    url - /api/v1/users 
    method - GET 
    IP - ::1
   {"timestamp":"2022-07-18T09:31:25.324Z"}
error: 
    status - 500 
    message - TypeError: Cannot read properties of undefined (reading 'email')
    at _callee$ (C:\Users\kanch\OneDrive\Desktop\FundooNotes\Login\src\services\/user.service.js:5:48)
    at tryCatch (C:\Users\kanch\OneDrive\Desktop\FundooNotes\Login\node_modules\@babel\runtime\helpers\regeneratorRuntime.js:86:17)
    at Generator._invoke (C:\Users\kanch\OneDrive\Desktop\FundooNotes\Login\node_modules\@babel\runtime\helpers\regeneratorRuntime.js:66:24)
    at Generator.next (C:\Users\kanch\OneDrive\Desktop\FundooNotes\Login\node_modules\@babel\runtime\helpers\regeneratorRuntime.js:117:21)
    at asyncGeneratorStep (C:\Users\kanch\OneDrive\Desktop\FundooNotes\Login\node_modules\@babel\runtime\helpers\asyncToGenerator.js:3:24)
    at _next (C:\Users\kanch\OneDrive\Desktop\FundooNotes\Login\node_modules\@babel\runtime\helpers\asyncToGenerator.js:25:9)
    at C:\Users\kanch\OneDrive\Desktop\FundooNotes\Login\node_modules\@babel\runtime\helpers\asyncToGenerator.js:32:7
    at new Promise (<anonymous>)
    at Object.<anonymous> (C:\Users\kanch\OneDrive\Desktop\FundooNotes\Login\node_modules\@babel\runtime\helpers\asyncToGenerator.js:21:12)
    at Object.getAllUsers (C:\Users\kanch\OneDrive\Desktop\FundooNotes\Login\src\services\/user.service.js:4:25)
    at _callee$ (C:\Users\kanch\OneDrive\Desktop\FundooNotes\Login\src\controllers\/user.controller.js:12:36)
    at tryCatch (C:\Users\kanch\OneDrive\Desktop\FundooNotes\Login\node_modules\@babel\runtime\helpers\regeneratorRuntime.js:86:17)
    at Generator._invoke (C:\Users\kanch\OneDrive\Desktop\FundooNotes\Login\node_modules\@babel\runtime\helpers\regeneratorRuntime.js:66:24)
    at Generator.next (C:\Users\kanch\OneDrive\Desktop\FundooNotes\Login\node_modules\@babel\runtime\helpers\regeneratorRuntime.js:117:21)
    at asyncGeneratorStep (C:\Users\kanch\OneDrive\Desktop\FundooNotes\Login\node_modules\@babel\runtime\helpers\asyncToGenerator.js:3:24)
    at _next (C:\Users\kanch\OneDrive\Desktop\FundooNotes\Login\node_modules\@babel\runtime\helpers\asyncToGenerator.js:25:9) 
    url - /api/v1/users 
    method - GET 
    IP - ::1
   {"timestamp":"2022-07-18T09:31:47.134Z"}
error: 
    status - 500 
    message - ReferenceError: email is not defined
    at _callee$ (C:\Users\kanch\OneDrive\Desktop\FundooNotes\Login\src\services\/user.service.js:5:36)
    at tryCatch (C:\Users\kanch\OneDrive\Desktop\FundooNotes\Login\node_modules\@babel\runtime\helpers\regeneratorRuntime.js:86:17)
    at Generator._invoke (C:\Users\kanch\OneDrive\Desktop\FundooNotes\Login\node_modules\@babel\runtime\helpers\regeneratorRuntime.js:66:24)
    at Generator.next (C:\Users\kanch\OneDrive\Desktop\FundooNotes\Login\node_modules\@babel\runtime\helpers\regeneratorRuntime.js:117:21)
    at asyncGeneratorStep (C:\Users\kanch\OneDrive\Desktop\FundooNotes\Login\node_modules\@babel\runtime\helpers\asyncToGenerator.js:3:24)
    at _next (C:\Users\kanch\OneDrive\Desktop\FundooNotes\Login\node_modules\@babel\runtime\helpers\asyncToGenerator.js:25:9)
    at C:\Users\kanch\OneDrive\Desktop\FundooNotes\Login\node_modules\@babel\runtime\helpers\asyncToGenerator.js:32:7
    at new Promise (<anonymous>)
    at Object.<anonymous> (C:\Users\kanch\OneDrive\Desktop\FundooNotes\Login\node_modules\@babel\runtime\helpers\asyncToGenerator.js:21:12)
    at Object.getAllUsers (C:\Users\kanch\OneDrive\Desktop\FundooNotes\Login\src\services\/user.service.js:4:25)
    at _callee$ (C:\Users\kanch\OneDrive\Desktop\FundooNotes\Login\src\controllers\/user.controller.js:12:36)
    at tryCatch (C:\Users\kanch\OneDrive\Desktop\FundooNotes\Login\node_modules\@babel\runtime\helpers\regeneratorRuntime.js:86:17)
    at Generator._invoke (C:\Users\kanch\OneDrive\Desktop\FundooNotes\Login\node_modules\@babel\runtime\helpers\regeneratorRuntime.js:66:24)
    at Generator.next (C:\Users\kanch\OneDrive\Desktop\FundooNotes\Login\node_modules\@babel\runtime\helpers\regeneratorRuntime.js:117:21)
    at asyncGeneratorStep (C:\Users\kanch\OneDrive\Desktop\FundooNotes\Login\node_modules\@babel\runtime\helpers\asyncToGenerator.js:3:24)
    at _next (C:\Users\kanch\OneDrive\Desktop\FundooNotes\Login\node_modules\@babel\runtime\helpers\asyncToGenerator.js:25:9) 
    url - /api/v1/users 
    method - GET 
    IP - ::1
   {"timestamp":"2022-07-18T09:32:54.832Z"}
error: 
    status - 500 
    message - TypeError: Cannot read properties of undefined (reading 'email')
    at _callee$ (C:\Users\kanch\OneDrive\Desktop\FundooNotes\Login\src\services\/user.service.js:5:48)
    at tryCatch (C:\Users\kanch\OneDrive\Desktop\FundooNotes\Login\node_modules\@babel\runtime\helpers\regeneratorRuntime.js:86:17)
    at Generator._invoke (C:\Users\kanch\OneDrive\Desktop\FundooNotes\Login\node_modules\@babel\runtime\helpers\regeneratorRuntime.js:66:24)
    at Generator.next (C:\Users\kanch\OneDrive\Desktop\FundooNotes\Login\node_modules\@babel\runtime\helpers\regeneratorRuntime.js:117:21)
    at asyncGeneratorStep (C:\Users\kanch\OneDrive\Desktop\FundooNotes\Login\node_modules\@babel\runtime\helpers\asyncToGenerator.js:3:24)
    at _next (C:\Users\kanch\OneDrive\Desktop\FundooNotes\Login\node_modules\@babel\runtime\helpers\asyncToGenerator.js:25:9)
    at C:\Users\kanch\OneDrive\Desktop\FundooNotes\Login\node_modules\@babel\runtime\helpers\asyncToGenerator.js:32:7
    at new Promise (<anonymous>)
    at Object.<anonymous> (C:\Users\kanch\OneDrive\Desktop\FundooNotes\Login\node_modules\@babel\runtime\helpers\asyncToGenerator.js:21:12)
    at Object.getAllUsers (C:\Users\kanch\OneDrive\Desktop\FundooNotes\Login\src\services\/user.service.js:4:25)
    at _callee$ (C:\Users\kanch\OneDrive\Desktop\FundooNotes\Login\src\controllers\/user.controller.js:12:36)
    at tryCatch (C:\Users\kanch\OneDrive\Desktop\FundooNotes\Login\node_modules\@babel\runtime\helpers\regeneratorRuntime.js:86:17)
    at Generator._invoke (C:\Users\kanch\OneDrive\Desktop\FundooNotes\Login\node_modules\@babel\runtime\helpers\regeneratorRuntime.js:66:24)
    at Generator.next (C:\Users\kanch\OneDrive\Desktop\FundooNotes\Login\node_modules\@babel\runtime\helpers\regeneratorRuntime.js:117:21)
    at asyncGeneratorStep (C:\Users\kanch\OneDrive\Desktop\FundooNotes\Login\node_modules\@babel\runtime\helpers\asyncToGenerator.js:3:24)
    at _next (C:\Users\kanch\OneDrive\Desktop\FundooNotes\Login\node_modules\@babel\runtime\helpers\asyncToGenerator.js:25:9) 
    url - /api/v1/users 
    method - GET 
    IP - ::1
   {"timestamp":"2022-07-18T10:03:25.299Z"}
error: 
    status - 500 
    message - Error:  User is not exist
    at _callee$ (C:\Users\kanch\OneDrive\Desktop\FundooNotes\Login\src\services\/user.service.js:9:11)
    at tryCatch (C:\Users\kanch\OneDrive\Desktop\FundooNotes\Login\node_modules\@babel\runtime\helpers\regeneratorRuntime.js:86:17)
    at Generator._invoke (C:\Users\kanch\OneDrive\Desktop\FundooNotes\Login\node_modules\@babel\runtime\helpers\regeneratorRuntime.js:66:24)
    at Generator.next (C:\Users\kanch\OneDrive\Desktop\FundooNotes\Login\node_modules\@babel\runtime\helpers\regeneratorRuntime.js:117:21)
    at asyncGeneratorStep (C:\Users\kanch\OneDrive\Desktop\FundooNotes\Login\node_modules\@babel\runtime\helpers\asyncToGenerator.js:3:24)
    at _next (C:\Users\kanch\OneDrive\Desktop\FundooNotes\Login\node_modules\@babel\runtime\helpers\asyncToGenerator.js:25:9)
    at processTicksAndRejections (node:internal/process/task_queues:96:5) 
    url - /api/v1/users/login 
    method - POST 
    IP - ::1
   {"timestamp":"2022-07-18T10:13:51.439Z"}
error: 
      status - 11000
      message - E11000 duplicate key error collection: practice.users index: email_1 dup key: { email: "manisha@gmail.com" } 
      url - /api/v1/users 
      method - POST 
      IP - ::1
     {"timestamp":"2022-07-19T01:59:27.830Z"}
error: 
    status - 500 
    message - RangeError: Invalid status code: 11000
    at new NodeError (node:internal/errors:371:5)
    at ServerResponse.writeHead (node:_http_server:274:11)
    at ServerResponse.writeHead (C:\Users\kanch\OneDrive\Desktop\FundooNotes\Login\node_modules\on-headers\index.js:44:26)
    at ServerResponse._implicitHeader (node:_http_server:265:8)
    at write_ (node:_http_outgoing:766:9)
    at ServerResponse.end (node:_http_outgoing:855:5)
    at ServerResponse.send (C:\Users\kanch\OneDrive\Desktop\FundooNotes\Login\node_modules\express\lib\response.js:232:10)
    at ServerResponse.json (C:\Users\kanch\OneDrive\Desktop\FundooNotes\Login\node_modules\express\lib\response.js:278:15)
    at appErrorHandler (C:\Users\kanch\OneDrive\Desktop\FundooNotes\Login\src\middlewares\/error.middleware.js:36:26)
    at Layer.handle_error (C:\Users\kanch\OneDrive\Desktop\FundooNotes\Login\node_modules\express\lib\router\layer.js:71:5)
    at trim_prefix (C:\Users\kanch\OneDrive\Desktop\FundooNotes\Login\node_modules\express\lib\router\index.js:326:13)
    at C:\Users\kanch\OneDrive\Desktop\FundooNotes\Login\node_modules\express\lib\router\index.js:286:9
    at Function.process_params (C:\Users\kanch\OneDrive\Desktop\FundooNotes\Login\node_modules\express\lib\router\index.js:346:12)
    at Immediate.next (C:\Users\kanch\OneDrive\Desktop\FundooNotes\Login\node_modules\express\lib\router\index.js:280:10)
    at Immediate._onImmediate (C:\Users\kanch\OneDrive\Desktop\FundooNotes\Login\node_modules\express\lib\router\index.js:646:15)
    at processImmediate (node:internal/timers:468:21) 
    url - /api/v1/users 
    method - POST 
    IP - ::1
   {"timestamp":"2022-07-19T01:59:27.849Z"}
error: 
    status - 500 
    message - ValidationError: "firstName" is required 
    url - /api/v1/users 
    method - POST 
    IP - ::1
   {"timestamp":"2022-07-19T02:09:38.608Z"}
error: 
    status - 500 
    message - ValidationError: "firstName" is required 
    url - /api/v1/users 
    method - POST 
    IP - ::1
   {"timestamp":"2022-07-19T02:10:02.989Z"}
